test/sqladmin-fetcher.ts

```diff
@@ -247,7 +248,7 @@ t.test('getEphemeralCertificate', async t => {
     ephemeralCert,
     {
       cert: CLIENT_CERT,
-      expirationTime: 'Jul 22 17:53:09 3022 GMT',
+      expirationTime: '3022-07-22T17:53:09.000Z',
```

Comment:
interesting, this seems unrelated to the auth type changes, why did it needed to change?

Reply:
This is because we're using `toISOString()` before we return the expiration time now.

Reply:
oh I see, yeah that makes sense!